src/main/_docs/admin-guide/admin-runbook.md

```diff
@@ -32,22 +33,206 @@ Zookeeper is a key-value store that is needed by Swarm in a clustered Codenvy se
 -A INPUT -m state --state NEW -m udp -p udp --dport 4789 -j ACCEPT
 ```
 
-**Step 2**: Create directories for Zookeeper:
+**Step 2**: Change the configuration of the Docker daemon to look like the below. On CentOS this file is found at `/etc/sysconfig/docker-network`:
 
 ```
-mkdir -p var/lib/zookeeper/datalog
+DOCKER_NETWORK_OPTIONS=' --bip=172.17.42.1/16 -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock --cluster-store=zk://<%= scope.lookupvar('third_party::docker::install::docker_cluster_store') -%>:2181 --cluster-advertise=<%= scope.lookupvar('third_party::docker::install::docker_cluster_advertise') -%>:2375'
```

Comment:
Need to change puppet variable to something that will represent master host URL
`<%= scope.lookupvar('third_party::docker::install::docker_cluster_store')  -%>` change to `codenvy-stg.com` or corresponding value for customer
Also  `<%= scope.lookupvar('third_party::docker::install::docker_cluster_advertise') -%>`, it need to be replaced with `eth0` on centos